https://rt.cpan.org/Public/Bug/Display.html?id=148415

Use of apostrophe as package separator was deprecated in Perl 5.37.9,
and will be removed in Perl 5.42.0.  You should change code that uses
Test::More::isn't() to use Test::More::isnt() as a replacement at t/07_stack.t line 20
Subject: 	Data-Clone-0.004-Fix-for-perl-5.38.patch
Data-Clone-0.004-Fix-for-perl-5.38.patch
diff -up Data-Clone-0.004/t/07_stack.t.orig Data-Clone-0.004/t/07_stack.t
--- Data-Clone-0.004/t/07_stack.t.orig	2023-05-23 15:51:30.526714411 +0200
+++ Data-Clone-0.004/t/07_stack.t	2023-05-23 15:51:43.223831287 +0200
@@ -17,6 +17,6 @@ use Data::Clone;
 
 my $before = bless [], Bar::;
 my $after  = clone($before);
-isn't $after, $before, 'stack reallocation during callback';
+isnt $after, $before, 'stack reallocation during callback';
 
 done_testing;
